java/templates/tests/test-service.yaml

```
@@ -12,4 +12,11 @@ spec:
       - name: SERVICE_NAME
         value: {{ template "hmcts.java.releaseName" . }}
     command: ["sh", "-c", "httpstatuscode=$(wget -S http://$SERVICE_NAME/health 2>&1 | grep HTTP/ | awk 'END{print $2}') && [ \"$httpstatuscode\" = \"200\" ]"]
-  restartPolicy: Never
\ No newline at end of file
+    resources:
+      limits:
+        cpu: 2500m
```

these limits seem very excessive
```suggestion
        cpu: 500m
```

---

done